packages/ui-default/index.ts

```
@@ -53,7 +53,7 @@ class LegacyModeHandler extends Handler {
 
   @param('legacy', Types.Boolean)
   @param('nohint', Types.Boolean)
-  async get(domainId: string, legacy = false, nohint = false) {
+  async get({ }, legacy = false, nohint = false) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)